### ISSUE TITLE
Abandoned project

### ISSUE THREAD

**Issue by ClassyCircuit** [geaz/sharpDox#98]
Is this project abandoned? There has been no activity for 2 years now..

**Comment by geaz** [comment]
Yes, this project is not longer maintained (should make it clear on in the readme).

I started a .net core rewrite of sharpDox (coreDox), but then Microsoft released DocFX (https://dotnet.github.io/docfx/). Thats why I, and others, thought, that there is no need anymore for sharpDox.
